src/index.html

```diff
@@ -1,15 +1,110 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
-</html>
+  <html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      <title>HTML Form</title>
+      <link rel="stylesheet" href="./style.css">
+    </head>
+    <body>
+      <form class="form">
+        <fieldset class="form-box">
+          <legend>Personal information</legend>
+          <label for="surname" class="form-box__item">
+            Surname:
+            <input id="surname" type="text" autocomplete="on" name="surname" required>
+          </label>
+
+          <label for="name" class="form-box__item">
+            Name: <input id="name" type="text" autocomplete="on" name="name" required>
+          </label>
+
+          <label for="number" class="form-box__item">
+            How old are You?
+            <input id="number" min="1" max="100" class="num" type="number" name="age" value="12">
```

Comment:
```suggestion
            <input id="number" min="1" max="100" type="number" name="age" value="12">
```
зараз в тебе вже немає цього класу